consensus/babe/types.go

```diff
@@ -45,10 +47,22 @@ type AuthorityData struct {
 
 // BabeHeader as defined in Polkadot RE Spec, definition 5.10 in section 5.1.4
 type BabeHeader struct {
-	VRFOutput          [32]byte
-	VRFProof           [32]byte
+	VrfOutput          [32]byte
+	VrfProof           [64]byte
 	BlockProducerIndex uint64
-	Slot               uint64
+	SlotNumber         uint64
+}
+
+func (bh *BabeHeader) Encode() []byte {
```

Comment:
What encoding is this? Please add comment.

Reply:
scale but in a custom func since it doesn't work with known size byte arrays :/